website_form_recaptcha/models/website_form_recaptcha.py

```diff
@@ -17,13 +17,13 @@ class WebsiteFormRecaptcha(models.AbstractModel):
     URL = 'https://www.google.com/recaptcha/api/siteverify'
     RESPONSE_ATTR = 'g-recaptcha-response'
     ERROR_MAP = {
-        'missing-input-secret': 'The secret parameter is missing.',
+        'missing-input-secret': _('The secret parameter is missing.'),
```

Comment:
You cannot translate strings outside a method because there's no `self.env.lang` variable to get the desired language, so if you're translating this (which is good IMHO), then do not use a class variable and use a method instead please 😊

Reply:
Good catch

Reply:
TIL